frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/pages/employee/LOI/ViewLOI.js

```diff
@@ -1,24 +1,128 @@
 import React, { useState, Fragment, useEffect, useRef } from "react";
-import {
-    Header,
-} from "@egovernments/digit-ui-react-components";
+import { Header, SubmitBar, Menu, ActionBar } from "@egovernments/digit-ui-react-components";
 import { useParams, useHistory } from "react-router-dom";
+import { Controller, useForm, useWatch } from 'react-hook-form'
 import { useTranslation } from "react-i18next";
 import { ApplicationDetailsTemplate } from "../../../../../templates/ApplicationDetails"
 import ApplicationDetailsContent from "../../../../../templates/ApplicationDetails/components/ApplicationDetailsContent";
-
+import ProcessingModal from "../../../components/Modal/ProcessingModal";
+import RejectLOIModal from '../../../components/Modal/RejectLOIModal'
 const ViewLOI = (props) => {
     const { t } = useTranslation()
-
+    const { register, control, watch, handleSubmit, formState: { errors, ...rest }, reset, trigger, getValues} = useForm({defaultValues: {}, mode: "onSubmit"});
+    const menuRef = useRef();
+    const [displayMenu, setDisplayMenu] = useState(false);
     let { isLoading, isError, data: applicationDetails, error } = Digit.Hooks.works.useViewLOIDetails(t);
+    
+    const tenantId = Digit.ULBService.getCurrentTenantId();
+    let paginationParams = { limit: 10, offset:0, sortOrder:"ASC" }
+    const { isLoading: hookLoading, data:employeeData } = Digit.Hooks.hrms.useHRMSSearch(
+        null,
+        tenantId,
+        paginationParams,
+        null
+    );
+    const { isLoading:desgLoading, data:designationData } = Digit.Hooks.useCustomMDMS(
+        "pb",
+        "common-masters",
+        [
+            {
+                "name": "Designation"
+            }
+        ]
+        );
+
+    if (designationData?.[`common-masters`]) {
+        var { Designation } = designationData?.[`common-masters`]
+    }
+    
+    const {  data, isFetched } = Digit.Hooks.useCustomMDMS(
+        "pb",
```

Comment:
Dont hardcode state tenant , there were many occurance for the same, @burhanj-eGov  can u check ?